### ISSUE TITLE
Tooltip not working on current values

### ISSUE THREAD

**Issue by satheesh786** [n3-charts/line-chart#293]
When i pointed my tooltip it just showing only first and last values but not middle values.

here is my code 

 <md-card class="charts" >
                    <md-card-content >
                        <span class="x-axis">Date</span>
                        <span class="y-axis">Measurements(mg/dL)</span>
                        <linechart  data="data" options="options" height="400"></linechart>
                    </md-card-content>
                </md-card>
![tooltip](https://cloud.githubusercontent.com/assets/12515306/9218165/871aa076-40ed-11e5-81a4-1b5a76df7c47.png)

        $scope.renderChart = function () {

            angular.forEach($scope.Bloodglucose, function (bg) {
                bg.measured = new Date(bg.measureddate);
            });
            console.log($scope.Bloodglucose);
            $scope.data = $scope.Bloodglucose;

            $scope.options = {
                lineMode: 'cardinal',
                axes: {
                    x: {
                        type: "date",
                        key: "measured",
                        ticksFormat: "%Y/%m/%d"
                    },
                    y: {
                        type: "linear"
                    }
                },
                series: [
                    {
                        y: "measurement",
                        type: 'area',
                        label: "Blood Glucose",
                        color: "#3498db",
                        dotSize: 3
                    }

                ],

                tooltip: {
                    mode: "scrubber",
                    formatter: function (x, y, series) {
                        return moment(x).format("MMM Do YY") + ': ' + y;
                    }
                }
       };

        };

**Comment by satheesh786** [comment]
In this ,when i pointed mouse on date 17th ,it is showing data values of date 13th.
The main problem is only first and last data values are showing .no middle data working on tooltip

**Comment by chaosmail** [comment]
Hi @satheesh786 

the scrubber tooltip "snaps" to the nearest data point relative to your mouse position. That's why you see the srucbber on the 13th while hovering over the 17th.

I agree that it might be useful to turn off this snappy behavior in the options.

Best,
Christoph

**Comment by satheesh786** [comment]
Even when i placed mode is none or axes ,tooltip not working.
I want to show exact value in tooltip ,when i hoverd.

**Comment by chaosmail** [comment]
Yes, also in tooltip mode `axes` the tooltip "snaps" to the nearest value. It is a tooltip rather than a cursor

**Comment by satheesh786** [comment]
Actually it was showing only starting values  and ending value .Tooltip showing these two values only .

**Comment by chaosmail** [comment]
Now I see your problem, sorry for the confusion.
Could you make a plnkr/CodePen to reproduce your problem?

**Comment by satheesh786** [comment]
Sorry ,it's not possible in codepen ,Iam bind my data from backend.So there is a data problem

**Comment by ledikari** [comment]
encountered the same problem, satheesh if you want to post your data in codepen, just create a mock data on your server side. Just look for data scope.

**Comment by dorranbekker** [comment]
I had the same issue, it seems to be caused by the time series not being oldest-to-newest. 
i.e. make sure your data starts with the oldest date and ends with the newest
